wemake_python_styleguide/visitors/ast/naming.py

```diff
@@ -33,6 +35,8 @@
 class _NameValidator(object):
     """Utility class to separate logic from the visitor."""
 
+    _unicode_pattern: ClassVar[Pattern] = re.compile(r'([a-zA-Z_0-9]+)')
```

Comment:
Why have you chosen this way instead of https://stackoverflow.com/questions/35889505/check-that-a-string-contains-only-ascii-characters/35890514 ?

Reply:
The implementation for tests a "class name" was not changed, because the new approach does not work with class name. Remove "_check_name" from "visit_classDef"?